contracts/Manager.sol

```diff
@@ -722,97 +429,179 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
     }
 
     /**
-     * @notice Revokes votes on already voted proposal.
-     * @param proposalId The ID of the proposal to vote on.
-     * @param index The index of the proposal ID in `dequeued`.
+     * @notice Computes the amount of stCELO that should be minted for a given
+     * amount of CELO deposited.
+     * @param celoAmount The amount of CELO deposited.
+     * @return The amount of stCELO that should be minted.
      */
-    function revokeVotes(uint256 proposalId, uint256 index) external {
-        IVote vote = IVote(voteContract);
+    function toStakedCelo(uint256 celoAmount) public view returns (uint256) {
+        uint256 stCeloSupply = stakedCelo.totalSupply();
+        uint256 celoBalance = account.getTotalCelo();
 
-        (uint256 totalYesVotes, uint256 totalNoVotes, uint256 totalAbstainVotes) = vote.revokeVotes(
-            msg.sender,
-            proposalId
-        );
+        if (stCeloSupply == 0 || celoBalance == 0) {
+            return celoAmount;
+        }
 
-        account.votePartially(proposalId, index, totalYesVotes, totalNoVotes, totalAbstainVotes);
+        return (celoAmount * stCeloSupply) / celoBalance;
     }
 
     /**
-     * @notice Unlock balance of vote stCelo and update beneficiary vote history.
-     * @param beneficiary The account to be unlocked.
+     * @notice Computes the amount of CELO that should be withdrawn for a given
+     * amount of stCELO burned.
+     * @param stCeloAmount The amount of stCELO burned.
+     * @return The amount of CELO that should be withdrawn.
      */
-    function updateHistoryAndReturnLockedStCeloInVoting(address beneficiary)
-        external
-        returns (uint256)
-    {
-        IVote vote = IVote(voteContract);
-        return vote.updateHistoryAndReturnLockedStCeloInVoting(beneficiary);
+    function toCelo(uint256 stCeloAmount) public view returns (uint256) {
+        uint256 stCeloSupply = stakedCelo.totalSupply();
+        uint256 celoBalance = account.getTotalCelo();
+
+        if (stCeloSupply == 0 || celoBalance == 0) {
+            return stCeloAmount;
+        }
+
+        return (stCeloAmount * celoBalance) / stCeloSupply;
     }
 
     /**
-     * @notice Unlock vote balance of stCelo.
-     * @param accountAddress The account to be unlocked.
+     * @notice Checks if strategy was depracated. Not allowed strategy is reverted to default.
```

Comment:
Maybe a different word rather than "deprecated", since we've been using that with a specific meaning in the context of the default strategy?